demo/sfml_opengl3/nuklear_sfml_gl3.h

```diff
@@ -442,7 +442,9 @@ nk_sfml_handle_event(sf::Event* evt)
         } else nk_input_motion(ctx, evt->touch.x, evt->touch.y);
         return 1;
     } else if(evt->type == sf::Event::TextEntered) {
-        nk_input_unicode(ctx, evt->text.unicode);
+		if (evt->text.unicode != 8) {  //this line will check user whether input BACKSPACE. 8 is the Unicode of BACKSPACE.  If true, the function will ignore BACKSPACE input.
```

Comment:
It seems this header file uses C89 comments (`/* */`) - I'd prefer obeying that though I won't require that. Also I think a comment `8 ~ backspace` would be enough (all the rest is visible from the source itself or 100% clear from the context in which this everything happens) :wink:.